index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>WebGPU Samples</title>
+    <meta
+      name="description"
+      content="The WebGPU Samples are a set of samples demonstrating the use of the WebGPU API."
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="favicon.ico"
+    />
+
+    <link href="css/styles.css" rel="stylesheet">
+
+  </head>
+  <script defer type="module" src="main.js"></script>
+  <body>
+    <div class="wrapper">
+      <nav class="panel container">
+        <h1>
+          <a href="./">WebGPU Samples</a>
+        </h1>
+        <input type="checkbox" id="menuToggle">
+        <label class="expand" for="menuToggle"></label>
+        <div class="panelContents">
+          <a href="https://github.com/webgpu/webpgu-samples">
+              Github
+          </a>
+          <hr>
+          <div id="samplelist"></div>
+          <hr>
+          <h3 style="margin-bottom: 5px">Other Pages</h3>
+          <ul class="exampleList">
+            <li>
+              <a
+                rel="noreferrer"
+                target="_blank"
+                href="/workload-simulator.html"
```

Comment:
I think this link should be:

```suggestion
                href="./workload-simulator.html"
```

Reply:
done